src/main/java/com/example/hibernate/controller/CarController.java

```diff
@@ -45,7 +45,7 @@ public ResponseEntity<Car> getCarById(@PathVariable("id") UUID id) {
         }
     }
 
-    @PostMapping("cars")
+    @PostMapping("/cars")
     public ResponseEntity<Car> createCar(@RequestBody Car car) {
```

Comment:
Controller probably should accept `AddCarRequest` or only `AddCar` because `Car` is a model with ID and etc, and you don't need to send that info.